README.md

```diff
@@ -65,7 +65,7 @@ Example CircleCI `config.yml`:
 version: 2.1
 
 orbs:
-  salus: federacy/salus@2.4.2
+  salus: federacy/salus@2.5.0
```

Comment:
did a new orb get deployed already?

Reply:
[Doesn't look like it, yet.](https://circleci.com/orbs/registry/orb/federacy/salus)

Reply:
Not yet. @jsulinski how do we deploy new orbs?

Reply:
It's bound to a CircleCI account. I'll deploy a new version now!

Reply:
Added a PR here: https://github.com/coinbase/salus/pull/64

Reply:
Published.

Reply:
Happy to add anyone that wants to publish. Alternatively, we could use `latest` as the default tag to remove the need to publish pegged versions.

Reply:
Yeah let's do latest as default!

I would prefer to have one less step to maintain!

Reply:
Okay, I'll make that change this weekend!

Reply:
PR for this is here: https://github.com/coinbase/salus/pull/66. Orb has already been pushed, so starting with 2.5.1 it will always pull latest unless you specify a custom version for an executor.